backend/src/database/libraries.go

```diff
@@ -7,17 +7,20 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func (db *DB) GetAllLibraries(page, perPage int, showHidden bool, search string, providerId int) (int64, []models.Library, error) {
+func (db *DB) GetAllLibraries(page, perPage int, visibility string, search string, providerId int) (int64, []models.Library, error) {
 	var libraries []models.Library
 	var total int64
 	offset := (page - 1) * perPage
-	tx := db.Model(&models.Library{})
-	if !showHidden {
+	tx := db.Model(&models.Library{}).Preload("OpenContentProvider").Order("created_at DESC")
+	if visibility == "hidden" {
+		tx = tx.Where("visibility_status = false")
+	}
+	if visibility == "visible" {
```

Comment:
I'm not 100% sure, but because this field isn't passed in from a user input but rather its passed in as a selectable (so there cant be any variation since its passed in from the `enum FilterLibrariesAdmin`), I don't think we need to convert to lowercase? But if you think I should add it I definitely can no problem!

Reply:
Let's add it to be safe.